### PR TITLE
[nrf noup] Select Experimental for persistent subscriptions

### PR DESCRIPTION
**Pull request by ArekBalysNordic**
Currently, there is an issue with persistent subscriptions when multiple controllers create subscriptions.

### CODE REVIEW

File: config/nrfconnect/chip-module/Kconfig.features [hunk 1]
```diff
@@ -290,4 +290,9 @@ config CHIP_LAST_FABRIC_REMOVED_ACTION_DELAY
 	  an action chosen by the CHIP_LAST_FABRIC_REMOVED_ACTION option. This schedule will allow for
 	  avoiding race conditions before the device removes non-volatile data.
 
+config CHIP_PERSISTENT_SUBSCRIPTIONS
```
Comment (Damian-Nordic) [inline config/nrfconnect/chip-module/Kconfig.features:293]: Why is this in `Kconfig.features` and not `Kconfig`?

**Review by LuDuda — APPROVED** [review]

File: config/nrfconnect/chip-module/Kconfig.features (hunk 1 above)
Comment (LuDuda) [reply config/nrfconnect/chip-module/Kconfig.features:293]: I thought about it as well when reviewing but the config/Zephyr where this is placed is generic for more platforms now. But I'm fine with adding experimental flag there - anyway.

Comment (ArekBalysNordic) [reply config/nrfconnect/chip-module/Kconfig.features:293]: It is the Matter feature and it is optional AFAIR, but I don't have a strong opinion on where it should be added. :)

Comment (ArekBalysNordic) [reply config/nrfconnect/chip-module/Kconfig.features:293]: Do you think that it fits better to Kconfig?

Comment (ArekBalysNordic) [reply config/nrfconnect/chip-module/Kconfig.features:293]: Ok, I see, indeed we may do something with that to make it more intuitive :)

**Review by Damian-Nordic — APPROVED** [review]

**Comment by ArekBalysNordic** [comment]
Unit Test error is not related to this PR.

File: config/nrfconnect/chip-module/Kconfig.features (hunk 1 above)
Comment (LuDuda) [reply config/nrfconnect/chip-module/Kconfig.features]: Cool, i misdunderstood, that we want to move it to `Zephyr` folder. Thanks, all makes sense.

**Pull request merged**